\hypertarget{fcexceptions_8hpp}{}\doxysection{src/fcexceptions.hpp 文件参考}
\label{fcexceptions_8hpp}\index{src/fcexceptions.hpp@{src/fcexceptions.hpp}}
{\ttfamily \#include $<$exception$>$}\newline
{\ttfamily \#include \char`\"{}fcenums.\+hpp\char`\"{}}\newline
fcexceptions.\+hpp 的引用(Include)关系图\+:
\nopagebreak
\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[width=236pt]{df/d9f/fcexceptions_8hpp__incl}
\end{center}
\end{figure}
此图展示该文件直接或间接的被哪些文件引用了\+:
\nopagebreak
\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[width=350pt]{df/d31/fcexceptions_8hpp__dep__incl}
\end{center}
\end{figure}
\doxysubsection*{类}
\begin{DoxyCompactItemize}
\item 
class \mbox{\hyperlink{classfc_runtime_exception}{fc\+Runtime\+Exception}}
\item 
class \mbox{\hyperlink{classfc_runtime_error}{fc\+Runtime\+Error}}
\item 
class \mbox{\hyperlink{classfc_calc_exception}{fc\+Calc\+Exception}}
\item 
class \mbox{\hyperlink{classfc_division_zero_exception}{fc\+Division\+Zero\+Exception}}
\item 
class \mbox{\hyperlink{classfc_syntax_exception}{fc\+Syntax\+Exception}}
\item 
class \mbox{\hyperlink{classfc_array_index_out_of_bound_exception}{fc\+Array\+Index\+Out\+Of\+Bound\+Exception}}
\end{DoxyCompactItemize}
\doxysubsection*{类型定义}
\begin{DoxyCompactItemize}
\item 
typedef std\+::exception \mbox{\hyperlink{fcexceptions_8hpp_a1ef7e03c22e6fee717ac6498e42a716c}{fcexception}}
\end{DoxyCompactItemize}


\doxysubsection{类型定义说明}
\mbox{\Hypertarget{fcexceptions_8hpp_a1ef7e03c22e6fee717ac6498e42a716c}\label{fcexceptions_8hpp_a1ef7e03c22e6fee717ac6498e42a716c}} 
\index{fcexceptions.hpp@{fcexceptions.hpp}!fcexception@{fcexception}}
\index{fcexception@{fcexception}!fcexceptions.hpp@{fcexceptions.hpp}}
\doxysubsubsection{\texorpdfstring{fcexception}{fcexception}}
{\footnotesize\ttfamily typedef std\+::exception \mbox{\hyperlink{fcexceptions_8hpp_a1ef7e03c22e6fee717ac6498e42a716c}{fcexception}}}



在文件 \mbox{\hyperlink{fcexceptions_8hpp_source}{fcexceptions.\+hpp}} 第 \mbox{\hyperlink{fcexceptions_8hpp_source_l00007}{7}} 行定义.

